Kconfig

```diff
@@ -1,65 +1,65 @@
 menu "Camera configuration"
-      
-config OV2640_SUPPORT
-	bool "OV2640 Support"
-	default y
-	help
-		Enable this option if you want to use the OV2640.
-		Disable this option to save memory.
 
-config OV7725_SUPPORT
-	bool "OV7725 Support"
-	default n
-	help
-		Enable this option if you want to use the OV7725.
-		Disable this option to save memory.
-	
-config OV3660_SUPPORT
-    bool "OV3660 Support"
+    config OV2640_SUPPORT
```

Comment:
wrong indentation? the `config` command should always be less indented than it's content (looks ok further down)